docs/en/observability/cloud-monitoring/aws/monitor-amazon-s3.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[monitor-amazon-s3]]
+= Monitor Amazon Simple Storage Service (S3)
+
+++++
+<titleabbrev>Monitor S3</titleabbrev>
+++++
+
+https://aws.amazon.com/s3/[Amazon Simple Storage Service (S3)]
+is a highly available object storage service that provides durability, security,
+and scalability. To store data, you create one or more buckets that contain
+objects. An object consists of a file and optional metadata that describes the
+file. For each bucket, you can control access to it.
+
+Like most AWS services, Amazon S3 and Amazon CloudWatch are integrated so you
+can collect, view, and analyze CloudWatch metrics for your S3 buckets to help
+understand and improve the performance of applications that use Amazon S3.
+The Elastic {integrations-docs}/aws/s3[Amazon S3 integration] collects metrics
+from Amazon CloudWatch using {agent}.
+
+With the Amazon S3 integration, you can collect these S3 metrics from
+CloudWatch:
+
+* Daily storage metrics for buckets. Use these metrics to monitor bucket
+storage. These metrics are reported once per day by default and are provided to
+AWS customers at no additional cost.
+* Request metrics. Use these metrics to quickly identify and act on
+operational issues. These request metrics are available at one-minute intervals
+after some latency for processing, and they are not enabled by default. 
+
+[discrete]
+[[get-started-s3]]
+== Get started
+
+If you plan to collect request metrics, enable them for the S3 buckets you want
+to monitor. To learn how, refer to the
+https://docs.aws.amazon.com/AmazonS3/latest/userguide/configure-request-metrics-bucket.html[AWS documentation].
+
+To collect S3 metrics, you typically need to install the Elastic
+{integrations-docs}/aws/s3[Amazon S3 integration] and deploy an {agent}
+locally or on an EC2 instance.
+
+Expand the **quick guide** to learn how, or skip to the next section if your
+data is already in {es}.
+
+//Sets the integration name used in the quick guide
+:integration-name: Amazon S3
+
+include::{observability-docs-root}/docs/en/shared/integrations-quick-guide.asciidoc[]
+
+//Clears the integration name
+:integration-name!:
+
+{agent} is currently the preferred way to add S3 logs and metrics. For other
```

Comment:
same here

Reply:
I think it's probably OK to mention that Elastic Agent can collect logs and metrics, but I'll remove the mention of logs in case it's confusing to users.